### ISSUE TITLE
Guide for developing in IntelliJ

### ISSUE THREAD

**Issue by evelinec** [OpenLiberty/guides-common#70]
Introductory guide on developing applications on Open Liberty in IntelliJ.  Should cover the use of Liberty build plugins in IntelliJ.

**Comment by yeekangc** [comment]
@meswan @jj-zhang are looking at this.

**Comment by turkeylurkey** [comment]
Guide is copied to the test server and the open liberty repository has been created.

**Comment by gkwan-ibm** [comment]
- ready for dev content review

**Comment by gkwan-ibm** [comment]
- dev content review in progress

**Comment by ankitagrawa** [comment]
Addressed the dev content feedback.

**Comment by ankitagrawa** [comment]
Peer review in progress.

**Comment by ankitagrawa** [comment]
- Addressing the feedback. 
- Peer review on hold.

**Comment by ankitagrawa** [comment]
Feedback addressed. Ready for peer review.
Jimmy - Mac
Nimrah - windows
Karthik - Linux

**Comment by ankitagrawa** [comment]
addressing peer review feedback.

**Comment by gkwan-ibm** [comment]
@jj-zhang 
- the 2nd dev content review feedback
  - https://github.com/OpenLiberty/draft-guide-intellij/issues/22

**Comment by gkwan-ibm** [comment]
- addressed the 2nd dev content review

**Comment by ankitagrawa** [comment]
addressing Trevors feedback.

**Comment by gkwan-ibm** [comment]
@jj-zhang 
- the 3rd dev content review feedback
  - https://github.com/OpenLiberty/draft-guide-intellij/issues/30

**Comment by gkwan-ibm** [comment]
- addressing feedback from Mike and the 3rd dev content review

**Comment by gkwan-ibm** [comment]
@jj-zhang 
- the 4th dev content review feedback
  - https://github.com/OpenLiberty/draft-guide-intellij/issues/33

**Comment by gkwan-ibm** [comment]
Peer review 
- Mac (Nirmah)
- Linux 
- Win (Danny)

**Comment by gkwan-ibm** [comment]
- ready for SME review

**Comment by gkwan-ibm** [comment]
- got SME feedback and need to discuss how to address them

**Comment by gkwan-ibm** [comment]
- created a video
- updating the guide for the link and some content

**Comment by ankitagrawa** [comment]
video pending on review.

**Comment by gkwan-ibm** [comment]
@jj-zhang 
- video feedback at https://github.com/OpenLiberty/draft-guide-intellij/issues/42

**Comment by ankitagrawa** [comment]
addressing the feedback.

**Comment by gkwan-ibm** [comment]
- rewriting the README

**Comment by gkwan-ibm** [comment]
- ready for dev content review

**Comment by gkwan-ibm** [comment]
- dev content review feedback
  - https://github.com/OpenLiberty/draft-guide-intellij/issues/45

**Comment by gkwan-ibm** [comment]
- working on the design
- addressing the dev content review feedback

**Comment by gkwan-ibm** [comment]
- the design is ready for review

**Comment by gkwan-ibm** [comment]
- addressing design review feedback

**Comment by yeekangc** [comment]
Guide isn't the best way to cover what we have for tools. They are best handled via built-in help or documentation plus blog posts (for latest enhancements etc). Closing.
